cluster/examples/kubernetes/ceph/operator-openshift.yaml

```diff
@@ -356,6 +356,9 @@ data:
   # Whether to start the discovery daemon to watch for raw storage devices on nodes in the cluster.
   # This daemon does not need to run if you are only going to create your OSDs based on StorageClassDeviceSets with PVCs.
   ROOK_ENABLE_DISCOVERY_DAEMON: "false"
+  # Enable volume replication controller
+  CSI_ENABLE_VOLUME_REPLICATION: "true"
+  CSI_VOL_REPLICATION_IMAGE: "quay.io/kubestorage/volumereplication-operator:latest"
```

Comment:
new image org `https://quay.io/repository/csiaddons/volumereplication-operator` need to be updated on other places also